R/make_vaccineff_data.R

```diff
@@ -203,40 +203,77 @@ summary.vaccineff_data <- function(object, warnings_log = FALSE, ...) {
       checkmate::test_class(object, "vaccineff_data")
```

Comment:
Same comment as before, is there ever a case where this code can run without the `object` being of the `<vaccineff_data>` class?

Instead you may want to validate the class, i.e. check it has all the elements you'd expect in a `<vaccineff_data>` object, instead of checking if it has the correct class attribute.

Reply:
I thought this was a common practice, but you're right. Solved in f1893fa4ee5d94be03c448ced898f95d6711854d